cmd/helpMessages.go

```diff
@@ -30,100 +30,122 @@ Copies source data to a destination location. The supported directions are:
 
-Upload a single file with a SAS:
   - azcopy cp "/path/to/file.txt" "https://[account].blob.core.windows.net/[container]/[path/to/blob]?[SAS]"
 
-Upload a single file with a SAS using piping (block blobs only):
+Upload a single file by using a SAS token and piping (block blobs only):
+  
   - cat "/path/to/file.txt" | azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/blob]?[SAS]"
 
-Upload an entire directory with a SAS:
+Upload an entire directory by using a SAS token:
+  
   - azcopy cp "/path/to/dir" "https://[account].blob.core.windows.net/[container]/[path/to/directory]?[SAS]" --recursive=true
 or
   - azcopy cp "/path/to/dir" "https://[account].blob.core.windows.net/[container]/[path/to/directory]?[SAS]" --recursive=true --put-md5
 
-Upload a set of files with a SAS using wildcards:
+Upload a set of files by using a SAS token and wildcard (*) characters:
+ 
   - azcopy cp "/path/*foo/*bar/*.pdf" "https://[account].blob.core.windows.net/[container]/[path/to/directory]?[SAS]"
 
-Upload files and directories with a SAS using wildcards:
+Upload files and directories by using a SAS token and wildcard (*) characters:
+
   - azcopy cp "/path/*foo/*bar*" "https://[account].blob.core.windows.net/[container]/[path/to/directory]?[SAS]" --recursive=true
 
-Download a single file using OAuth authentication. Please use 'azcopy login' command first if you aren't logged in yet:
+Download a single file by using OAuth authentication. If you have not yet logged into AzCopy, please use azcopy login command before you run the following command.
+
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/blob]" "/path/to/file.txt"
 
-Download a single file with a SAS:
+Download a single file by using a SAS token:
+
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/blob]?[SAS]" "/path/to/file.txt"
 
-Download a single file with a SAS using piping (block blobs only):
+Download a single file by using a SAS token and then piping the output to a file (block blobs only):
+  
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/blob]?[SAS]" > "/path/to/file.txt"
 
-Download an entire directory with a SAS:
+Download an entire directory by using a SAS token:
+  
   - azcopy cp "https://[account].blob.core.windows.net/[container]/[path/to/directory]?[SAS]" "/path/to/dir" --recursive=true
 
-A note about wildcards in URLs:
+A note about using a wildcard character (*) in URLs:
 
-The only usage of a wildcard in a URL that is supported is the final, trailing /*, and in the container name when not specifying a blob name.
-If you happen to use the character * in the name of a blob, please manually encode it to %2A to avoid it being treated as a wildcard character.
+There's only two supported ways to use a wildcard character in a URL. 
+You can use one just after the final forward slash (/) of a URL to obtain all of the files in a directory. 
```

Comment:
Reword this to something along the lines of
"A trailing wildcard (in the form of /*) places all files under the directory immediately at the destination instead of under a sub-directory"

Currently, the sentence seems ambiguous, as though --recursive does the same thing (when really it does not)